[2025-04-16T03:53:04.325+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-16T03:53:04.337+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: schedule_data_master.expurgo_clientes manual__2025-04-16T03:52:58.712158+00:00 [queued]>
[2025-04-16T03:53:04.344+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: schedule_data_master.expurgo_clientes manual__2025-04-16T03:52:58.712158+00:00 [queued]>
[2025-04-16T03:53:04.344+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-16T03:53:04.358+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): expurgo_clientes> on 2025-04-16 03:52:58.712158+00:00
[2025-04-16T03:53:04.360+0000] {standard_task_runner.py:72} INFO - Started process 322 to run task
[2025-04-16T03:53:04.363+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'schedule_data_master', 'expurgo_clientes', 'manual__2025-04-16T03:52:58.712158+00:00', '--job-id', '2203', '--raw', '--subdir', 'DAGS_FOLDER/dag_master.py', '--cfg-path', '/tmp/tmp7k2oly4f']
[2025-04-16T03:53:04.365+0000] {standard_task_runner.py:105} INFO - Job 2203: Subtask expurgo_clientes
[2025-04-16T03:53:04.383+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.8/site-packages/airflow/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-16T03:53:04.407+0000] {task_command.py:467} INFO - Running <TaskInstance: schedule_data_master.expurgo_clientes manual__2025-04-16T03:52:58.712158+00:00 [running]> on host 14cd87e75c31
[2025-04-16T03:53:04.474+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='schedule_data_master' AIRFLOW_CTX_TASK_ID='expurgo_clientes' AIRFLOW_CTX_EXECUTION_DATE='2025-04-16T03:52:58.712158+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-16T03:52:58.712158+00:00'
[2025-04-16T03:53:04.475+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-16T03:53:04.482+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-04-16T03:53:04.484+0000] {subprocess.py:88} INFO - Running command: ['/bin/bash', '-c', 'docker exec linux_python /bin/bash -c \'/workspace/scripts/expurgo_server.sh /path/to/move_cloud/clientes/ && echo "Script Executado com Sucesso!"\'']
[2025-04-16T03:53:04.494+0000] {subprocess.py:99} INFO - Output:
[2025-04-16T03:53:04.574+0000] {subprocess.py:106} INFO - Verificando diretorio: /path/to/move_cloud/clientes/
[2025-04-16T03:53:04.574+0000] {subprocess.py:106} INFO - Conteúdo do diretório antes da limpeza:
[2025-04-16T03:53:04.574+0000] {subprocess.py:106} INFO - clientes.csv
[2025-04-16T03:53:04.574+0000] {subprocess.py:106} INFO - Apagando itens do diretório...
[2025-04-16T03:53:04.575+0000] {subprocess.py:106} INFO - Verificando se o diretório está vazio...
[2025-04-16T03:53:04.575+0000] {subprocess.py:106} INFO - O diretório está vazio.
[2025-04-16T03:53:04.575+0000] {subprocess.py:106} INFO - Script Executado com Sucesso!
[2025-04-16T03:53:04.579+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-04-16T03:53:04.600+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-16T03:53:04.601+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=schedule_data_master, task_id=expurgo_clientes, run_id=manual__2025-04-16T03:52:58.712158+00:00, execution_date=20250416T035258, start_date=20250416T035304, end_date=20250416T035304
[2025-04-16T03:53:04.629+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-16T03:53:04.646+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-16T03:53:04.647+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
